scripts/nuget.autopkg

```diff
@@ -1,3 +1,11 @@
+configurations {
+      	// This node contains custom pivot information.
+      	Toolset 
+      	{
+           	key: "PlatformToolset";
+           	choices: { v140, v120, v110, v100 };
```

Comment:
I'm not very familiar with NuGet - this list has 4 versions of VS but below only 2 of them have associated files - does this list have to have just v140 and v120?